src/shared/components/results/styles/render-column-in-card.scss

```
@@ -0,0 +1,11 @@
+@import '../../../../../node_modules/franklin-sites/src/styles/settings';
+
+.result-card__info-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.result-card__info-bit:not(:last-child)::after {
+  content: ' · ';
```

```suggestion
  content: '·';
```
This looks a bit off on my screen when using spaces (a bit too much to the right), whereas not using spaces centres it perfectly for me.